assets/queries/openAPI/oauth_invalid_authorization_url/query.rego

```diff
@@ -0,0 +1,47 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	object.get(doc, "components", "undefined") != "undefined"
```

Comment:
Maybe lines 9 and 10 are unnecessary. 'sec_scheme := doc.components.securitySchemes[key]' already check if attributes components and securitySchemes are defined. Same for lines 13, 31, 32 and 35.